src/components/prescriptions/PrescriptionAddComponent.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+    <div
+        class="prescriptionBox"
+        @submit.prevent="addTest">
+        <h3 class="prescriptionBox__header">Dodaj receptę:</h3>
+        <form class="prescriptionBox__form prescriptionForm">
+            <div>
+                <div class="inputBox">
+                    <label for="nameAndSurnameDoctor">
+                        Imię i nazwisko lekarza:</label
+                    >
```

Comment:
add please ID for element which you want to be described by this label. It is important according to accessibility.